### ISSUE TITLE
Reorder items under "ABOUT [SYSTEM NAME]" in VAMC menus to match CAIA guidance

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#16233]
## User Story or Problem Statement
When getting IA guidance for Police Transparency, we also got a [request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/67055#issuecomment-1819824600) to standardize the order of VAMC menu items in the ABOUT [SYSTEM NAME] sub menu.

ABOUT [SYSTEM NAME]
- About us
- Programs (if enabled)
- Research (if enabled)
- Policies (or [Privacy and policies](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16234))
- VA police
- Work with us
- Contact us

[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMklWQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2310d46996b8681531fd373a1651fbd3668d238c/Screenshot%202023-11-28%20153011.png)

## Acceptance Criteria
- [x] Menu items are in the order specified
- [x] Submenu items follow their parent accordingly

**Comment by swirtSJW** [comment]
police and policies on adjacent lines is not great for scanning and recognition as they only differ by an 'i' (least visibly distinct character there is)

**Comment by swirtSJW** [comment]
Related upstream module  issue https://www.drupal.org/project/codit_menu_tools/issues/3406650

**Comment by xiongjaneg** [comment]
ready for Drupal refinement, will need Dave P. to review along the way

**Comment by swirtSJW** [comment]
This is a bit of a pain.  Some of our menus don't follow the pattern of "____-health-care" because the names are too long :(

**Comment by swirtSJW** [comment]
Upstream issues completed:
- [Rename MenuInsert class to MenuManipulator](https://www.drupal.org/project/codit_menu_tools/issues/3420289)
- [Add way to separate items in a menu](https://www.drupal.org/project/codit_menu_tools/issues/3406863)
- [Add way to separate items in a menu](https://www.drupal.org/project/codit_menu_tools/issues/3406863)

I will create a new release of the module in the morning after one final round of testing.

**Comment by swirtSJW** [comment]
There are some menu naming anomolies that I am fixing as I find them.  Noting them here as I go.

1.  Altoona ->  Contact Us changed to Contact us.

**Comment by swirtSJW** [comment]
This is an odd one. 
![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/5752113/9c4c453d-67b9-4ba6-b252-a0705470aec4)

The menu does exist with that machine name But it is empty  /admin/structure/menu/manage/va-leavenworth-health-care

I am guessing this was an old one that got renamed.

**Comment by swirtSJW** [comment]
Yep Leavenworth was coalesced to kansas and the menu was never removed.  https://github.com/department-of-veterans-affairs/va.gov-cms/issues/4943

I'll have to remove the menu from config as part of this PR :(

**Comment by swirtSJW** [comment]
Menus that are already correct, to be used to validate they don't break:

- There are none in the right order  :(


Menus that are currently out of order:

- /washington-dc-health-care
- /spokane-health-care
- /greater-los-angeles-health-care

**Comment by swirtSJW** [comment]
Running the script (CMS Devops) and fixing Lovell by hand (Facilities engineer) are covered by https://github.com/department-of-veterans-affairs/va.gov-cms/issues/17202

**Comment by swirtSJW** [comment]
The initial script was run twice on prod but kept failing due to running out of memory.  #17236 Should solve the problem so that #17202 can be completed.

**Comment by jilladams** [comment]
Have pinged on #17202 to clarify whether the CMS Team has run the required script in prod yet or not. If so: this ticket will be unblocked.

**Comment by xiongjaneg** [comment]
Sort of a duplicate of #17202, which was verified as completed in prod as following the menu order
<img width="362" alt="Screenshot 2024-03-06 at 10 01 39 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/d301d633-c3e0-4590-bb66-ae6cc779f79d">
<img width="362" alt="Screenshot 2024-03-06 at 10 01 52 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/ee984c38-2a5a-403b-a4ed-a77a206638b3">
<img width="312" alt="Screenshot 2024-03-06 at 10 02 11 AM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/73648238/eba509ed-2997-4c37-b197-ba9a9df09cfe">
